locales/en/kafka.json

```diff
@@ -109,6 +109,13 @@
   "partitions": "Partitions",
   "partitions_value": "up to 500",
   "select_filter_category": "Select filter category",
+  "settings": {
+    "connection_re_authentication_label": "Connection re-authentication:",
+    "disable": "Disable",
```

Comment:
Assuming this is the label for the submit button in the prompt, can you change this to "Turn off"?

Chatting with the CCS team, they recommend against using "Disable" for the prompt:

- IBM Style Guide also recommends against disable in this case: 
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#d
- Everywhere else, we use the language "Turn off", "Turn on" or some variation of this